### ISSUE TITLE
Graves and other mods reqests

### ISSUE THREAD

**Issue by slowiu06** [kreezxil/better-than-minecolonies-getting-started#2]
1. Grave/corpse/deathchest mod, any will be good.
2. https://www.curseforge.com/minecraft/mc-mods/xaeros-world-map - it should be in the modpack.
3. https://www.curseforge.com/minecraft/mc-mods/zmod-new-skills-system - mining matters, fighting too.
4. https://www.curseforge.com/minecraft/mc-mods/endergetic - because end is so empty.
5. Vanilla+ Tools mod is useless right now, when there is Silent Gems.
6. https://www.curseforge.com/minecraft/mc-mods/friendly-fire - feeling bad when my puppies dies from my hand.
7. https://www.curseforge.com/minecraft/mc-mods/pet-bats - I was dreaming about bat army.
8. https://www.curseforge.com/minecraft/mc-mods/reap-mod - replanting crops is boring.
9. https://www.curseforge.com/minecraft/mc-mods/trample-stopper - ^^^
10. https://www.curseforge.com/minecraft/mc-mods/atomicstrykers-infernal-mobs or something similar - make mobs more challenging.
11. https://www.curseforge.com/minecraft/mc-mods/must-go-faster or something like this - let rest my spacebar a little.

~Best wishes
Alchemik/Alchemiczny

**Comment by kreezxil** [comment]
1 will not be added, grave mods make any pack too easy. I spent over 9 months on World of Dragons wondering why players kept saying it was too easy. Didn't make sense. because I had dragons in it and lots of other mean mobs. it was grave mods causing that condition. Players were fearless and quickly were able to possess O.P. level stuff because they never lost anything. Grave mods break immersion.

**if you don't like that** you are welcome to add one to your own copy of the pack.

Xaero's Minimap has been added to todays patch.

I thought I was done, but I see i got some more to do, thanks for opening this.
